### PR TITLE
feat: improve and update pallet-move's benchmarks

### PR DESCRIPTION
**Pull request by neutrinoks**
- improved pallet-move's benchmarks by updating gas costs and adding more Move-scripts
- updated Substrate-weights after updating the benchmarks

### CODE REVIEW

File: pallet/src/weights.rs [hunk 1]
```diff
@@ -38,52 +38,52 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
...
 	}
 	/// Storage: `MoveModule::VMStorage` (r:1 w:1)
 	/// Proof: `MoveModule::VMStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn publish_module_bundle() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `111`
 		//  Estimated: `3576`
-		// Minimum execution time: 161_000_000 picoseconds.
-		Weight::from_parts(168_000_000, 0)
+		// Minimum execution time: 158_000_000 picoseconds.
```
Comment (Rqnsom) [inline pallet/src/weights.rs:85]: We can discuss this one on Slack

File: pallet/src/benchmarking.rs [hunk 2]
```diff
@@ -1,155 +1,216 @@
 //! Benchmarking setup for pallet-move.
 
-use frame_benchmarking::benchmarks;
-#[cfg(test)]
-use frame_benchmarking::impl_benchmark_test_suite;
+use frame_benchmarking::v2::*;
+// #[cfg(test)]
+// use frame_benchmarking::v2::test;
```
Comment (Rqnsom) [inline pallet/src/benchmarking.rs:5]: Could be removed then

**Review by Rqnsom — APPROVED** [review]
LGTM

File: pallet/src/weights.rs (hunk 1 above)
Comment (neutrinoks) [reply pallet/src/weights.rs]: Implemented a parameter-based benchmarking for this extrinsic 👍

File: pallet/src/benchmarking.rs (hunk 2 above)
Comment (neutrinoks) [reply pallet/src/benchmarking.rs]: Done

**Review by Rqnsom — APPROVED** [review]
LGTM 👍

**Pull request merged**